### ISSUE TITLE
Additional details of how this data was compiled

### ISSUE THREAD

**Issue by ccammilleri** [golem445/Corporate_Masks#1]
Great idea! You clearly understand that corporate environments do not equal consumer facing products and services and have different commonly used passwords. Nice work!

Is there a blog post to go along with how the dataset was handled to compile these results? I had a few questions after reviewing the outputs:

1. Are the mask files order by most prevalent or by efficiency. I'm very surprised that the number one mask for 14 char is all decimal. which leads to...
2. How far back does the dataset go, 1990, 2006, 2019?
3. Is this including machine and service accounts?
4. Are you able to say how many unique organizations were included in the dataset?

Again, great work!

**Comment by golem445** [comment]
Thanks! I intended to respond sooner, but that corona life though haha. In truth, this post garnered more interest than expected. Maybe a blog post would be a good idea.
1. Good eye. Mask lists are separated for efficiency, however (hat tip to roycewilliams for heads up) a statsgen file was later added for anyone desiring to sort by popularity.
2) Datasets runs from 2015-2020. (Password policies before 2015 steadily decreased in quality and seemed less helpful)
3) Some service accounts, no machine accounts. (I've only cracked a few machine accounts, but they were all 127 characters)
4) Wish I could friend :) 

Hope that helps!
